src/lwc/gsApplicationStatus/gsApplicationStatus.js

```diff
@@ -0,0 +1,130 @@
+import getApplicationStatus from '@salesforce/apex/GS_ApplicationStatusController.getApplicationStatus'
+import gsNoApplicationSubmitted from '@salesforce/label/c.gsNoApplicationSubmitted'
+import gsLearnMore from '@salesforce/label/c.gsLearnMore'
+import gsDaysRemainingInFreeTrial from '@salesforce/label/c.gsDaysRemainingInFreeTrial'
+import gsApplyForFreeLicenses from '@salesforce/label/c.gsApplyForFreeLicenses'
+import gsApplicationStatus from '@salesforce/label/c.gsApplicationStatus'
+import gsSubmitted from '@salesforce/label/c.gsSubmitted'
+import gsDaysAdded from '@salesforce/label/c.gsDaysAdded'
+import gsCheckStatus from '@salesforce/label/c.gsCheckStatus'
+import gsApplicationStatusModalHeader from '@salesforce/label/c.gsApplicationStatusModalHeader'
+import gsClose from '@salesforce/label/c.commonClose'
+import gsFollowUpApplicationStatus from '@salesforce/label/c.gsFollowUpApplicationStatus'
+export default class GsApplicationStatus extends LightningElement {
+    
+    @track errorMessage = "";
+    @track diffInDays = null;
+    @track isApplicationSubmitted = false;
+    @track isLoading = false;
+    @track img = "";
+    @track isActiveInstance = false;
+    applyForFreeLicensesImg = Resources + '/gsResources/Accept_Tasks_Apply_Card.png';
+    checkForStatusImg = Resources + '/gsResources/gift_illustration_2.svg';
+
+    labels = {
+        gsNoApplicationSubmitted,
+        gsLearnMore,
+        gsDaysRemainingInFreeTrial,
+        gsApplyForFreeLicenses,
+        gsApplicationStatus,
+        gsSubmitted,
+        gsDaysAdded,
+        gsCheckStatus,
+        gsApplicationStatusModalHeader,
+        gsClose,
+        gsFollowUpApplicationStatus
+    }
+
+    /**
+     * Initialized the component with the data retrieved from Salesforce
+     */
+    connectedCallback() {
+        this.showSpinner();
+
+        getApplicationStatus()
+        .then(result => {
+            this.diffInDays = this.calculateTrialRemainingDays(result);
+            this.isApplicationSubmitted = this.checkApplicationSubmitted(result);
+            this.img = this.isApplicationSubmitted ?  this.checkForStatusImg : this.applyForFreeLicensesImg; 
+            this.isActiveInstance = result.trialExpirationDate == null;
+            this.hideSpinner();
+        })
+        .catch(error => {
+            this.errorMessage = error;
+            this.hideSpinner();
+        });
+    }
+
+    /**
+     * Shows the spinner in the component
+     */
+    showSpinner() {
+        this.isLoading = true;
+    }
+
+    /**
+     * Hides the spinner in the component
+     */
+    hideSpinner() {
+        this.isLoading = false;
+    }
+
+    /**
+     * 
+     * @param {Object} result - Retrieved result object from Salesforce, in order to this to work it has to have a 'trialExpirationDate' field.
+     * @returns remaining days between today and expiration day or -1.
+     */
+    calculateTrialRemainingDays(result) {
```

Comment:
This does not work properly if the Trial Expiration has passed. The issue is that the logic is using abs to convert a negative value to positive value. The result is that my expired org shows as 4 days remaining instead of 4 days past the expiration date.

Example JSON:
```
{   
	"applicationDate": "2020-10-30",
	"isSandbox": true,
	"trialExpiration": "2020-11-22"	
}
```

Results in the UI showing the following:
![image](https://user-images.githubusercontent.com/801017/100265178-dcce5980-2f1d-11eb-8c3f-b1dd0430b236.png)

Reply:
I considered this case but I didnt want to do anything regarding this. The json overriding is for testing only and in real world this case is invalid(you cannot access a org which is expired). So as this scenario is invalid for testing I add no control on it